.test-infra/metrics/README.md

```diff
@@ -84,23 +95,31 @@ docker-compose build
 
 # Spinup docker-compose related containers.
 docker-compose up
+docker-compose -f docker-compose.prometheus.yml up
+
+# The above commands can be combined.
+docker-compose up -f docker-compose.yml -f docker-compose.prometheus.yml up
 ```
 
 After running these commands, you can access the services running on your local
 machine:
 
 * Grafana: http://localhost:3000
-* Postgres DB: localhost:5432
+* Postgres DB: http://localhost:5432
+* Prometheus: http://localhost:9090
+* Pushgateway: http://localhost:9091
+* Alertmanager: http://localhost:9093
 
 If you're deploying for the first time on your machine, follow the wiki instructions
 on how to manually [configure
 Grafana](https://cwiki.apache.org/confluence/display/BEAM/Community+Metrics#CommunityMetrics-GrafanaUI).
 
-Grafana and Postgres containers persist data to Docker volumes, which will be
+Grafana, Postgres and Prometheus containers persist data to Docker volumes, which will be
 restored on subsequent runs. To start from a clean state, you must also wipe out
 these volumes. (List volumes via `docker volume ls`)
 
 ## Kubernetes setup
 
-Kubernetes deployment instructions are maintained in the
-[wiki](https://cwiki.apache.org/confluence/display/BEAM/Community+Metrics).
+Kubernetes deployment instructions are maintained in the wiki:
+* [Community metrics](https://cwiki.apache.org/confluence/display/BEAM/Community+Metrics)
+* [Test metrics]() <!-- TODO: add a link to instructions -->
```

Comment:
Please, add link or create jira ticket for this. It is an important part of documentation we do not want to miss.

Reply:
I created jira ticket: https://issues.apache.org/jira/browse/BEAM-8163.